README.md

```diff
@@ -242,3 +244,5 @@ Thanks and have an awesome day 👋
 [issues]: https://github.com/Voyz/ibeam/issues
 
 [gateway]: https://interactivebrokers.github.io/cpwebapi/
+
+[IBC]: https://github.com/IbcAlpha/IBC
```

Comment:
just for consistency, I use all-lowercase-with-dashes for link variables. Could you update it?